meta/meta.go

```diff
@@ -296,6 +296,28 @@ func (m *Meta) CreateTable(dbID int64, tableInfo *model.TableInfo) error {
 	return m.txn.HSet(dbKey, tableKey, data)
 }
 
+// CreateView creates a view with tableInfo in database.
+func (m *Meta) CreateView(dbID int64, tableInfo *model.TableInfo, orReplace bool) error {
+	// Check if db exists.
+	dbKey := m.dbKey(dbID)
+	if err := m.checkDBExists(dbKey); err != nil {
+		return errors.Trace(err)
+	}
+
+	// Check if view exists.
+	tableKey := m.tableKey(tableInfo.ID)
+	if err := m.checkTableNotExists(dbKey, tableKey); err != nil && !orReplace {
```

Comment:
We do not need to check `orReplace` here since we've already checked it outside.

Reply:
Please address this comment.